webapp/src/components/NewsItem.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <h1>{{ title }} {{ (votes) }}</h1>
+    <button v-on:click="incrementVotes">Upvote</button>
+    <button v-on:click="decrementVotes">Downvote</button>
+    <button v-on:click="removeMe">Remove</button>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "NewsItem",
+  props:["title","votes"],
```

Comment:
```suggestion
  props: {
    title: { type: String, required: true },
    votes: { type: Number: required: true},
  }
```
It's good practice to declare data types of properties.